pkg/provider/azure_vmss.go

```diff
@@ -1026,7 +1026,6 @@ func (ss *ScaleSet) EnsureHostInPool(service *v1.Service, nodeName types.NodeNam
 		})
 	primaryIPConfiguration.LoadBalancerBackendAddressPools = &newBackendPools
 	newVM := &compute.VirtualMachineScaleSetVM{
-		Sku:      vm.Sku,
```

Comment:
Nothing required from here? only capacity was causing us problems.

Reply:
yes

Reply:
set `vmss.Sku` can cause capacity change? I still don't believe that, `vmss.Sku` should be const after vmss is set up, right?

Reply:
> vmss.Sku should be const after vmss is set up, right?

We think it is const initially, but we're wrong. The capacity of VMSS is part of Sku:

https://github.com/kubernetes-sigs/cloud-provider-azure/blob/2fe4926f176d49990e161d2296604f72cb2a6ce8/vendor/github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-12-01/compute/models.go#L9469-L9476

```